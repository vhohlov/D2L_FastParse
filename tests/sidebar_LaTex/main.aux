\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1} Course: Introduction }{2}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}Functional Programming }{7}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {2.1} Course: Side-effects and recursive functions }{7}{section.2.1}}
\@writefile{toc}{\contentsline {section}{ Course: Side-effects and recursive functions }{146}{lstnumber.-226.3}}
\@writefile{toc}{\contentsline {chapter}{\numberline {3}Logic programming}{147}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {3.0} Course: Side-effects and recursive functions }{180}{Item.41}}
\@writefile{toc}{\contentsline {section}{ Course: Side-effects and recursive functions }{181}{Item.41}}
\@writefile{toc}{\contentsline {chapter}{\numberline {4}Notes}{181}{chapter.4}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {4.0} Course: Side-effects and recursive functions }{181}{section*.2}}
