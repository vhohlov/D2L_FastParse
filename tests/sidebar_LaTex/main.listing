not(G) :- G,!,fail.
not(_).

