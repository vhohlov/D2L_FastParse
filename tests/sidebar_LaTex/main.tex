\documentclass[a4paper, 12pt]{report}
\usepackage[margin=2.54cm]{geometry}
%type of input encoding
\usepackage[utf8x]{inputenc}
%determines what font to use, not input encoding
%default OT1 supports 7-bit enconding chars
%T1 used for printing chars with 8-bit encodigns
\usepackage[T1]{fontenc}
%%for printing greek letters
\usepackage{textgreek}
\usepackage{titlesec}
\usepackage[titletoc, title]{appendix}
\titleformat{\chapter}
{\normalfont\LARGE\bfseries}{\thechapter}{1em}{}
\titleformat{\section}{\LARGE\bfseries}
\titleformat{\subsection}{\LARGE\bfseries}
\titleformat{\subsubsection}{\LARGE\bfseries}
\titleformat*{\section}{\LARGE\bfseries}
\titleformat*{\subsection}{\Large\bfseries}
\titleformat*{\subsubsection}{\large\bfseries}
%for printing greek letters
\usepackage{amsmath}
\usepackage{amsthm}
\usepackage{amsfonts}
\usepackage{ulem}
\usepackage[unicode]{hyperref}
\usepackage[table]{xcolor}
\usepackage{multirow}
\usepackage{graphicx}
\usepackage{tcolorbox}
%loads listings package for general use
%and also avaiable this way inside tcolorbox
\tcbuselibrary{listings}
%breaks a tcolorbox over pages
\tcbuselibrary{breakable}
%for printing theorems
\tcbuselibrary{theorems}%used for printing notes at the end of file
%continuous used for one numbering in the whole doc
\usepackage[continuous]{pagenote}
\makepagenote
\let\footnote=\pagenote
%for not counting Notes as a new section
%\renewcommand*{\notedivision}{\section*{\notesname}}
%for not printing sections divisions in Notes
\renewcommand*{\pagenotesubhead}[1]{}\graphicspath{ {./media/} }
\usepackage[export]{adjustbox}
\usepackage{graphicx}
\usepackage{wrapfig}
\setlength{\parindent}{0in}
%for warning Overfull \hbox
\hfuzz=50.002pt
%settings for listings env
\lstset{breaklines=true,
numbers=left,
numberstyle=\small,
numbersep=8pt,
xleftmargin=10pt,
escapeinside=&&
}
\newtcbtheorem{_def}{Definition}{breakable}{df}
\newtcbtheorem{prop}{Proposition}{breakable}{prp}
\newtcbtheorem{_proof}{Proof}{breakable}{prf}
\newtcbtheorem{remark}{Remark}{breakable}{rmk}
\newtcbtheorem{example}{Example}{breakable}{exp}
\newtcbtheorem{theorem}{Theorem}{breakable}{th}
\newtcbtheorem{exercise}{Exercise}{breakable}{exer}
\newtcbtheorem{solution}{Solution}{breakable}{sol}
\newtcbtheorem{algorithm}{Algorithm}{breakable}{alg}
\title{Un document Latex}
\author{Victor Hohlov}
\date{2018-05-05 }

\begin{document}
\maketitle
\tableofcontents
\chapter{ Course: Introduction }
\input{pp:intro}
\chapter{Functional Programming }
\section{ Course: Side-effects and recursive functions }
\input{pp:rec}
\subsection{ Lab: Introduction to Haskell}
\input{pp:l01}
\section{ Course: Higher-order functions}
\input{pp:high}
\subsection{ Lab: Higher-order functions}
\input{pp:l02}
\subsection{ Lab: Even more higher-order functions}
\input{pp:l03}
\section{ Course: Types in functional programming}
\input{pp:types}
\section{ Course: Abstract datatypes}
\input{pp:adt}
\subsection{ Lab: Abstract datatypes}
\input{pp:l04}
\section{ Course: Polymorphism in functional languages}
\input{pp:polymorphism}
\subsection{ Lab: Haskell type-classes}
\input{pp:l05}
\section{ Course: Parameter passing in Programming Languages}
\input{pp:c06}
\section{ Course: Lazy evaluation in Haskell}
\input{pp:c07}
\subsection{ Lab: Haskell and Scheme evaluation}
\input{pp:l06}
\section{ Course: The Lambda Calculus}
\input{pp:lambda}
\section{ H01: Haskell Lazy Dynamic Programming}
\input{pp:h01}
\chapter{Logic programming}
\section{ Course: Introduction to Prolog}
\input{pp:c08}
\subsection{ Lab: Prolog 101}
\input{pp:l07}
\section{ Course: Clauses, unification, cut, negation}
\input{pp:c09}
\subsection{ Lab: Prolog 102}
\input{pp:l08}
\subsection{ Lab: Prolog graphs}
\input{pp:l09}
\newpage
\printnotes
\end{document}\documentclass[a4paper, 12pt]{report}
\usepackage[margin=2.54cm]{geometry}
%type of input encoding
\usepackage[utf8x]{inputenc}
%determines what font to use, not input encoding
%default OT1 supports 7-bit enconding chars
%T1 used for printing chars with 8-bit encodigns
\usepackage[T1]{fontenc}
%%for printing greek letters
\usepackage{textgreek}
\usepackage{titlesec}
\usepackage[titletoc, title]{appendix}
\titleformat{\chapter}
{\normalfont\LARGE\bfseries}{\thechapter}{1em}{}
\titleformat*{\section}{\LARGE\bfseries}
\titleformat*{\subsection}{\Large\bfseries}
\titleformat*{\subsubsection}{\large\bfseries}
%for printing greek letters
\usepackage{amsmath}
\usepackage{amsthm}
\usepackage{amsfonts}
\usepackage{ulem}
\usepackage[unicode]{hyperref}
\usepackage[table]{xcolor}
\usepackage{multirow}
\usepackage{graphicx}
\usepackage{tcolorbox}
%loads listings package for general use
%and also avaiable this way inside tcolorbox
\tcbuselibrary{listings}
%breaks a tcolorbox over pages
\tcbuselibrary{breakable}
%for printing theorems
\tcbuselibrary{theorems}%used for printing notes at the end of file
%continuous used for one numbering in the whole doc
\usepackage[continuous]{pagenote}
\makepagenote
\let\footnote=\pagenote
%for not counting Notes as a new section
%\renewcommand*{\notedivision}{\section*{\notesname}}
%for not printing sections divisions in Notes
\renewcommand*{\pagenotesubhead}[1]{}\graphicspath{ {./media/} }
\usepackage[export]{adjustbox}
\usepackage{graphicx}
\usepackage{wrapfig}
\setlength{\parindent}{0in}
%for warning Overfull \hbox
\hfuzz=50.002pt
%settings for listings env
\lstset{breaklines=true,
numbers=left,
numberstyle=\small,
numbersep=8pt,
xleftmargin=10pt,
escapeinside=&&
}
\newtcbtheorem{_def}{Definition}{breakable}{df}
\newtcbtheorem{prop}{Proposition}{breakable}{prp}
\newtcbtheorem{_proof}{Proof}{breakable}{prf}
\newtcbtheorem{remark}{Remark}{breakable}{rmk}
\newtcbtheorem{example}{Example}{breakable}{exp}
\newtcbtheorem{theorem}{Theorem}{breakable}{th}
\newtcbtheorem{exercise}{Exercise}{breakable}{exer}
\newtcbtheorem{solution}{Solution}{breakable}{sol}
\newtcbtheorem{algorithm}{Algorithm}{breakable}{alg}
\title{Un document Latex}
\author{Victor Hohlov}
\date{2018-05-05 }

\begin{document}
\maketitle
\tableofcontents
\chapter{ Course: Introduction }
\input{pp:intro}
\chapter{Functional Programming }
\section{ Course: Side-effects and recursive functions }
\input{pp:rec}
\subsection{ Lab: Introduction to Haskell}
\input{pp:l01}
\section{ Course: Higher-order functions}
\input{pp:high}
\subsection{ Lab: Higher-order functions}
\input{pp:l02}
\subsection{ Lab: Even more higher-order functions}
\input{pp:l03}
\section{ Course: Types in functional programming}
\input{pp:types}
\section{ Course: Abstract datatypes}
\input{pp:adt}
\subsection{ Lab: Abstract datatypes}
\input{pp:l04}
\section{ Course: Polymorphism in functional languages}
\input{pp:polymorphism}
\subsection{ Lab: Haskell type-classes}
\input{pp:l05}
\section{ Course: Parameter passing in Programming Languages}
\input{pp:c06}
\section{ Course: Lazy evaluation in Haskell}
\input{pp:c07}
\subsection{ Lab: Haskell and Scheme evaluation}
\input{pp:l06}
\section{ Course: The Lambda Calculus}
\input{pp:lambda}
\section{ H01: Haskell Lazy Dynamic Programming}
\input{pp:h01}
\chapter{Logic programming}
\section{ Course: Introduction to Prolog}
\input{pp:c08}
\subsection{ Lab: Prolog 101}
\input{pp:l07}
\section{ Course: Clauses, unification, cut, negation}
\input{pp:c09}
\subsection{ Lab: Prolog 102}
\input{pp:l08}
\subsection{ Lab: Prolog graphs}
\input{pp:l09}
\newpage
\printnotes
\end{document}\documentclass[a4paper, 12pt]{report}
\usepackage[margin=2.54cm]{geometry}
%type of input encoding
\usepackage[utf8x]{inputenc}
%determines what font to use, not input encoding
%default OT1 supports 7-bit enconding chars
%T1 used for printing chars with 8-bit encodigns
\usepackage[T1]{fontenc}
%%for printing greek letters
\usepackage{textgreek}
\usepackage{titlesec}
\usepackage[titletoc, title]{appendix}
\titleformat{\chapter}
{\normalfont\LARGE\bfseries}{\thechapter}{1em}{}
\titleformat*{\section}{\LARGE\bfseries}
\titleformat*{\subsection}{\Large\bfseries}
\titleformat*{\subsubsection}{\large\bfseries}
%for printing greek letters
\usepackage{amsmath}
\usepackage{amsthm}
\usepackage{amsfonts}
\usepackage{ulem}
\usepackage[unicode]{hyperref}
\usepackage[table]{xcolor}
\usepackage{multirow}
\usepackage{graphicx}
\usepackage{tcolorbox}
%loads listings package for general use
%and also avaiable this way inside tcolorbox
\tcbuselibrary{listings}
%breaks a tcolorbox over pages
\tcbuselibrary{breakable}
%for printing theorems
\tcbuselibrary{theorems}%used for printing notes at the end of file
%continuous used for one numbering in the whole doc
\usepackage[continuous]{pagenote}
\makepagenote
\let\footnote=\pagenote
%for not counting Notes as a new section
%\renewcommand*{\notedivision}{\section*{\notesname}}
%for not printing sections divisions in Notes
\renewcommand*{\pagenotesubhead}[1]{}\graphicspath{ {./media/} }
\usepackage[export]{adjustbox}
\usepackage{graphicx}
\usepackage{wrapfig}
\setlength{\parindent}{0in}
%for warning Overfull \hbox
\hfuzz=50.002pt
%settings for listings env
\lstset{breaklines=true,
numbers=left,
numberstyle=\small,
numbersep=8pt,
xleftmargin=10pt,
escapeinside=&&
}
\newtcbtheorem{_def}{Definition}{breakable}{df}
\newtcbtheorem{prop}{Proposition}{breakable}{prp}
\newtcbtheorem{_proof}{Proof}{breakable}{prf}
\newtcbtheorem{remark}{Remark}{breakable}{rmk}
\newtcbtheorem{example}{Example}{breakable}{exp}
\newtcbtheorem{theorem}{Theorem}{breakable}{th}
\newtcbtheorem{exercise}{Exercise}{breakable}{exer}
\newtcbtheorem{solution}{Solution}{breakable}{sol}
\newtcbtheorem{algorithm}{Algorithm}{breakable}{alg}
\title{Un document Latex}
\author{Victor Hohlov}
\date{2018-05-05 }

\begin{document}
\maketitle
\tableofcontents
\chapter{ Course: Introduction }
\input{pp:intro}
\chapter{Functional Programming }
\section{ Course: Side-effects and recursive functions }
\input{pp:rec}
\subsection{ Lab: Introduction to Haskell}
\input{pp:l01}
\section{ Course: Higher-order functions}
\input{pp:high}
\subsection{ Lab: Higher-order functions}
\input{pp:l02}
\subsection{ Lab: Even more higher-order functions}
\input{pp:l03}
\section{ Course: Types in functional programming}
\input{pp:types}
\section{ Course: Abstract datatypes}
\input{pp:adt}
\subsection{ Lab: Abstract datatypes}
\input{pp:l04}
\section{ Course: Polymorphism in functional languages}
\input{pp:polymorphism}
\subsection{ Lab: Haskell type-classes}
\input{pp:l05}
\section{ Course: Parameter passing in Programming Languages}
\input{pp:c06}
\section{ Course: Lazy evaluation in Haskell}
\input{pp:c07}
\subsection{ Lab: Haskell and Scheme evaluation}
\input{pp:l06}
\section{ Course: The Lambda Calculus}
\input{pp:lambda}
\section{ H01: Haskell Lazy Dynamic Programming}
\input{pp:h01}
\chapter{Logic programming}
\section{ Course: Introduction to Prolog}
\input{pp:c08}
\subsection{ Lab: Prolog 101}
\input{pp:l07}
\section{ Course: Clauses, unification, cut, negation}
\input{pp:c09}
\subsection{ Lab: Prolog 102}
\input{pp:l08}
\subsection{ Lab: Prolog graphs}
\input{pp:l09}
\newpage
\printnotes
\end{document}