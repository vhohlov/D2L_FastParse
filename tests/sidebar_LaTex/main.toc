\contentsline {chapter}{\numberline {1} Course: Introduction }{2}{chapter.1}
\contentsline {chapter}{\numberline {2}Functional Programming }{7}{chapter.2}
\contentsline {section}{\numberline {2.1} Course: Side-effects and recursive functions }{7}{section.2.1}
\contentsline {section}{ Course: Side-effects and recursive functions }{146}{lstnumber.-226.3}
\contentsline {chapter}{\numberline {3}Logic programming}{147}{chapter.3}
\contentsline {section}{\numberline {3.0} Course: Side-effects and recursive functions }{180}{Item.41}
\contentsline {section}{ Course: Side-effects and recursive functions }{181}{Item.41}
\contentsline {chapter}{\numberline {4}Notes}{181}{chapter.4}
\contentsline {section}{\numberline {4.0} Course: Side-effects and recursive functions }{181}{section*.2}
