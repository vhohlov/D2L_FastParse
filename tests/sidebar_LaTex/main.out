\BOOKMARK [0][-]{chapter.1}{\376\377\000\040\000C\000o\000u\000r\000s\000e\000:\000\040\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n\000\040}{}% 1
\BOOKMARK [0][-]{chapter.2}{\376\377\000F\000u\000n\000c\000t\000i\000o\000n\000a\000l\000\040\000P\000r\000o\000g\000r\000a\000m\000m\000i\000n\000g\000\040}{}% 2
\BOOKMARK [1][-]{section.2.1}{\376\377\000\040\000C\000o\000u\000r\000s\000e\000:\000\040\000S\000i\000d\000e\000-\000e\000f\000f\000e\000c\000t\000s\000\040\000a\000n\000d\000\040\000r\000e\000c\000u\000r\000s\000i\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000s\000\040}{chapter.2}% 3
\BOOKMARK [1][-]{lstnumber.-226.3}{\376\377\000\040\000C\000o\000u\000r\000s\000e\000:\000\040\000S\000i\000d\000e\000-\000e\000f\000f\000e\000c\000t\000s\000\040\000a\000n\000d\000\040\000r\000e\000c\000u\000r\000s\000i\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000s\000\040}{chapter.2}% 4
\BOOKMARK [0][-]{chapter.3}{\376\377\000L\000o\000g\000i\000c\000\040\000p\000r\000o\000g\000r\000a\000m\000m\000i\000n\000g}{}% 5
\BOOKMARK [1][-]{Item.41}{\376\377\000\040\000C\000o\000u\000r\000s\000e\000:\000\040\000S\000i\000d\000e\000-\000e\000f\000f\000e\000c\000t\000s\000\040\000a\000n\000d\000\040\000r\000e\000c\000u\000r\000s\000i\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000s\000\040}{chapter.3}% 6
\BOOKMARK [1][-]{Item.41}{\376\377\000\040\000C\000o\000u\000r\000s\000e\000:\000\040\000S\000i\000d\000e\000-\000e\000f\000f\000e\000c\000t\000s\000\040\000a\000n\000d\000\040\000r\000e\000c\000u\000r\000s\000i\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000s\000\040}{chapter.3}% 7
\BOOKMARK [0][-]{chapter.4}{\376\377\000N\000o\000t\000e\000s}{}% 8
\BOOKMARK [1][-]{section*.2}{\376\377\000\040\000C\000o\000u\000r\000s\000e\000:\000\040\000S\000i\000d\000e\000-\000e\000f\000f\000e\000c\000t\000s\000\040\000a\000n\000d\000\040\000r\000e\000c\000u\000r\000s\000i\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000s\000\040}{chapter.4}% 9
